### PR TITLE
Dutch translation of README

### PR DESCRIPTION
**Pull request by timwienk**
README in Dutch! :netherlands:

I wasn't entirely sure where to add the Dutch document to the Translations table in the English README.md. Let me know if you're happy with this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,9 @@
       <td>
         <a href="/docs/README.tr.md">Türkçe</a>
       </td>
+      <td>
```
Comment (mfix22) [inline README.md:60]: Can you move this to be next to the "Add +" `td`? 🙂 

There is no real method to the ordering, besides just keeping them the order in which they have been translated

**Review by mfix22 — COMMENTED** [review]
This is great! Just one nit

Thanks @timwienk

File: README.md (hunk 1 above)
Comment (timwienk) [reply README.md:60]: Like this? (Swapped "Add +" and "Nederlands" around.)

Comment (mfix22) [reply README.md:60]: Works for me!

**Review by mfix22 — APPROVED** [review]

**Pull request merged**